engine/src/test/java/org/hibernate/validator/test/internal/constraintvalidators/hv/AgeValidatorTest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.test.internal.constraintvalidators.hv;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static org.hibernate.validator.testutil.ConstraintViolationAssert.assertNoViolations;
+import static org.hibernate.validator.testutil.ConstraintViolationAssert.assertThat;
+import static org.hibernate.validator.testutil.ConstraintViolationAssert.violationOf;
+import static org.hibernate.validator.testutils.ValidatorUtil.getConfiguration;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+import java.util.Set;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validator;
+
+import org.hibernate.validator.HibernateValidator;
+import org.hibernate.validator.HibernateValidatorConfiguration;
+import org.hibernate.validator.cfg.ConstraintMapping;
+import org.hibernate.validator.cfg.defs.AgeMinDef;
+import org.hibernate.validator.constraints.AgeMin;
+import org.hibernate.validator.constraintvalidation.HibernateConstraintValidator;
+import org.hibernate.validator.internal.constraintvalidators.hv.AgeMinValidator;
+import org.hibernate.validator.internal.util.annotation.ConstraintAnnotationDescriptor;
+import org.hibernate.validator.testutil.TestForIssue;
+
+import org.hibernate.validator.testutils.ConstraintValidatorInitializationHelper;
+import org.testng.annotations.Test;
+
+import java.time.LocalDate;
+
+/**
+ * A set of tests for {@link AgeMin} constraint validator ({@link AgeMinValidator}), which
+ * make sure that validation is performed correctly.
+ *
+ * @author Hillmer Chona
+ * @since 6.0.8
+ */
+@TestForIssue(jiraKey = "HV-1552" )
+public class AgeValidatorTest {
+
+	private int value = 18;
```

Comment:
Maybe make a constant of it and use in other occurrences of this number as well. WDYT?

Reply:
Done